src/app/home/home.page.ts

```diff
@@ -7,6 +8,23 @@ import { Component } from '@angular/core';
 })
 export class HomePage {
 
-  constructor() {}
+  currentImage:any ="./../../assets/18353.jpg";
+
+  constructor(private camera: Camera) {}
+
+  takePictures(){
+    const options: CameraOptions = {
+      quality: 100,
+      destinationType: this.camera.DestinationType.DATA_URL,
+      encodingType: this.camera.EncodingType.JPEG,
+      mediaType: this.camera.MediaType.PICTURE
+     };
+
+     this.camera.getPicture(options).then((imageData)=>{
+       this.currentImage = 'data:image/jpeg;base64,' + imageData;
+     }, (err) => {
+       console.log("Camera Issues: "+err);
+     });
```

Comment:
You should create a service for getting an image from the camera, as well as from the gallery. Then you can import this service into any component you need, and just call it on one line.

Reply:
I did that in the current request